src/d3a/d3a_core/simulation.py

```diff
@@ -149,6 +150,9 @@ def _init(self, slowdown, seed, paused, pause_after):
             log.info("Random seed: {}".format(random_seed))
 
         self.area = self.setup_module.get_setup(self.simulation_config)
+        if GlobalConfig.POWER_FLOW:
+            self.power_flow = PowerFlow(self.area)
+            self.power_flow.run_power_flow()
```

Comment:
I think you can write this in one line:
```
self.power_flow = PowerFlow(self.area).run_power_flow()
```

Reply:
`self.power_flow` is an instance variable that has to be used later to access other methods too, so I think your suggestion wont work.

Reply:
Yes, you are right! Thanks